pod/perlfunc.pod

```diff
@@ -7155,9 +7155,9 @@ X<select> X<filehandle, default>
 Returns the currently selected filehandle.  If FILEHANDLE is supplied,
 sets the new current default filehandle for output.  This has two
 effects: first, a L<C<write>|/write FILEHANDLE> or a L<C<print>|/print
-FILEHANDLE LIST> without a filehandle
-default to this FILEHANDLE.  Second, references to variables related to
-output will refer to this output channel.
+FILEHANDLE LIST> (or L<C<say>|/say FILEHANDLE LIST>) without a
```

Comment:
I'd add a comma before, so it reads: "a write, print, or say without a filehandle..."

Reply:
Force-pushed